pkg/local_object_storage/blobstor/blobovnicza.go

```diff
@@ -544,12 +542,18 @@ func (b *blobovniczas) getRangeFromLevel(prm *GetRangeSmallPrm, blzPath string,
 }
 
 // removes object from blobovnicza and returns DeleteSmallRes.
-func (b *blobovniczas) deleteObject(blz *blobovnicza.Blobovnicza, prm *blobovnicza.DeletePrm) (*DeleteSmallRes, error) {
+func (b *blobovniczas) deleteObject(blz *blobovnicza.Blobovnicza, prm *blobovnicza.DeletePrm, dp *DeleteSmallPrm) (*DeleteSmallRes, error) {
```

Comment:
`blobovnicza.DeletePrm` looks a bit useless after this change, but if you feel ok about it, I don't mind to keep it.

Reply:
It is accepted as parameter because we call `deleteObject` several times with the same args. The best way would be to keep all parameters in a single context structure, and pass it, changing it if necessary. But now it's easier to just pass an additional argument.